force-app/main/default/classes/CaseRoutingServiceTest.cls

```diff
@@ -0,0 +1,44 @@
+@IsTest
+public with sharing class CaseRoutingServiceTest {
+    @TestSetup static void setup() {
+        Case newTestCase = new Case();
+        newTestCase.OwnerId = '00G7Q000000Iz7eUAC';
+        newTestCase.Preferred_Language__c = 'English';
```

Comment:
All this values should not be plain strings. They should be defined as public static Final strings or enums in Utils class.